Source/Tools/CodeSizeValidation/Main.cpp

```diff
@@ -0,0 +1,432 @@
+#include "DummyHandlers.h"
+#include "FEXCore/Core/Context.h"
+#include "FEXCore/Debug/InternalThreadState.h"
+#include <FEXCore/Config/Config.h>
+#include <FEXCore/Utils/Allocator.h>
+#include <FEXCore/Utils/File.h>
+#include <FEXCore/Utils/FileLoading.h>
+#include <FEXCore/Utils/LogManager.h>
+
+namespace CodeSize {
+  class CodeSizeValidation final {
+    public:
+      struct InstructionStats {
+        uint64_t GuestCodeInstructions{};
+        uint64_t HostCodeInstructions{};
+
+        uint64_t HeaderSize;
```

Comment:
Should these be initialized to zero as well, like the members above?

Reply:
Zero initialized.